kythe/go/platform/vfs/vfs.go

```diff
@@ -87,6 +87,20 @@ type Writer interface {
 	Remove(ctx context.Context, path string) error
 }
 
+// FileReader composes interfaces from io that readable files from the vfs must
+// implement.
+type FileReader interface {
+	io.ReadCloser
+	io.ReaderAt
+	io.Seeker
+}
+
+// FileReader composes interfaces from io that writable files from the vfs must
+// implement.
+type FileWriter interface {
```

Comment:
This interface doesn't really seem to serve any purpose over just `io.WriteCloser`?

Reply:
Removed.